### ISSUE TITLE
path.charAt is not a function

### ISSUE THREAD

**Issue by internalfx** [paulmillr/chokidar#314]
Trying to track down a bug that popped up when chokidar updated to `1.0.4`, `1.0.3` works fine.

Any Ideas?

```
return path.charAt(0) === '/';
                ^
TypeError: path.charAt is not a function
  at posix (/Users/thomas/.nvm/versions/io.js/v2.2.1/lib/node_modules/quickshot/node_modules/chokidar/node_modules/path-is-absolute/index.js:4:14)
  at /Users/thomas/.nvm/versions/io.js/v2.2.1/lib/node_modules/quickshot/node_modules/chokidar/index.js:214:16
  at Array.map (native)
  at EventEmitter.FSWatcher._isIgnored (/Users/thomas/.nvm/versions/io.js/v2.2.1/lib/node_modules/quickshot/node_modules/chokidar/index.js:213:33)
  at EventEmitter.NodeFsHandler._addToNodeFs (/Users/thomas/.nvm/versions/io.js/v2.2.1/lib/node_modules/quickshot/node_modules/chokidar/lib/nodefs-handler.js:429:12)
  at EventEmitter.<anonymous> (/Users/thomas/.nvm/versions/io.js/v2.2.1/lib/node_modules/quickshot/node_modules/chokidar/index.js:424:12)
  at /Users/thomas/.nvm/versions/io.js/v2.2.1/lib/node_modules/quickshot/node_modules/chokidar/node_modules/async-each/index.js:16:7
  at Array.forEach (native)
  at each (/Users/thomas/.nvm/versions/io.js/v2.2.1/lib/node_modules/quickshot/node_modules/chokidar/node_modules/async-each/index.js:15:11)
  at EventEmitter.FSWatcher.add (/Users/thomas/.nvm/versions/io.js/v2.2.1/lib/node_modules/quickshot/node_modules/chokidar/index.js:423:5)
  at Object.exports.watch (/Users/thomas/.nvm/versions/io.js/v2.2.1/lib/node_modules/quickshot/node_modules/chokidar/index.js:500:33)
  at /Users/thomas/.nvm/versions/io.js/v2.2.1/lib/node_modules/quickshot/lib/theme/watch.js:101:32
```

**Comment by kerihenare** [comment]
@internalfx Are you passing a RegExp for `ignored`?

**Comment by internalfx** [comment]
Yes I am.

**Comment by internalfx** [comment]
Although it looks like you already found it :smile:

**Comment by es128** [comment]
Just published the fix in 1.0.5. Thanks for reporting.
